modules/@apostrophecms/image/ui/apos/components/AposMediaManagerDisplay.vue

```diff
@@ -132,13 +132,21 @@ export default {
   methods: {
     async uploadMedia (event) {
       this.$emit('upload-started');
+      const fileCount = event.target.files.length;
       const file = event.target.files[0];
 
       const emptyDoc = await apos.http.post(this.moduleOptions.action, {
         body: {
           _newInstance: true
         }
       });
+      await apos.notify(
+        // TODO: i18n
+        `Uploading ${fileCount} image${fileCount > 1 ? 's' : ''}`,
```

Comment:
I was about to tell you how to do this with %s placeholders and more arguments to apos.notify, but it involves pluralization and we don't actually have a wrapper for that yet. Hmm. OK for now.